lua/tokyonight/extra/fzf.lua

```diff
@@ -0,0 +1,18 @@
+local util = require("tokyonight.util")
+
+local M = {}
+
+--- @param colors ColorScheme
+function M.generate(colors)
+  return util.template(M.template, colors)
+end
+
+M.template = [[
+export FZF_DEFAULT_OPTS="$FZF_DEFAULT_OPTS \
+--color=fg:${fg},bg:${bg},hl:${orange} \ 
```

Comment:
I think the extra space at the end of line shouldn't be there. I missed it while reviewing this PR previously.

Reply:
Could you please elaborate why? See how `catppucin` does it https://github.com/catppuccin/fzf

Reply:
Because `\` is an escape character that will make you escape some special characters otherwise would cause errors. 

Basically, anything you put right after escape character is ignored so that you can add a new line (`\n`) without any problem. If you don't add `\` and add a new line, it will cause errors.

When you add a space after escape character, you will make it ignore that space character. And the new line character will become unescaped, which will cause errors because of unterminated string.

Reply:
>See how catppuccin does it https://github.com/catppuccin/fzf

I just checked how catppuccin does it. They are doing fine. No problem with that. You should delete the extra space at the end of line like this:

```suggestion
--color=fg:${fg},bg:${bg},hl:${orange} \
```